pkgs/development/python-modules/pip-system-certs/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, wheel
+, git-versioner
+, wrapt
+}:
+
+buildPythonPackage rec {
+  pname = "pip_system_certs";
```

```suggestion
  pname = "pip-system-certs";
```

And fix the `inherit pname` below.